dashboard/utils.py

```diff
@@ -176,7 +176,10 @@ def has_verified_cert(self, course_id):
         Returns:
             bool: whether the user has a verified cert meaning that they passed the course on edX
```

Comment:
This comment is no longer accurate. In fact, this function is named `has_verified_cert`, and that _name_ is no longer accurate. I would suggest renaming this function `has_downloadable_cert`.